Libraries/plist/CMakeLists.txt

```diff
@@ -69,6 +69,11 @@ add_executable(plutil Tools/plutil.cpp)
 target_link_libraries(plutil plist process util)
 install(TARGETS plutil DESTINATION usr/bin)
 
+add_executable(PlistBuddy Tools/PlistBuddy.cpp ${CMAKE_SOURCE_DIR}/ThirdParty/linenoise/linenoise.c)
+target_include_directories(PlistBuddy PRIVATE "${CMAKE_SOURCE_DIR}/ThirdParty/linenoise")
+target_link_libraries(PlistBuddy plist util)
+install(TARGETS PlistBuddy DESTINATION usr/bin)
```

Comment:
hmm, although this is convenient CLI feature, if this would cause problem, we can go without the library.
Just kept it in place since @jasonprado had put it in place before.